extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/CertificateConfig.java

```diff
@@ -81,6 +81,12 @@
     @ConfigItem
     public Optional<String> keyStoreKeyAlias;
 
+    /**
+     * An optional parameter to define the password for the key, in case it's different from {@link #keyStorePassword}.
```

Comment:
```suggestion
     * An optional parameter to define the password for the key, in case it's different from {@link #keyStorePassword}. To be used in combination with {@link #keyStoreAlias}.
```

Reply:
I had something like that. Same topic as in https://github.com/quarkusio/quarkus/pull/18387#discussion_r663721603 - I'll check!

Reply:
As outlined in my other comment, this is not required technically but most cases will come with both key-alias _and_ key-password.
So I'm not -1 on this change but I'd wait until that other discussion comes to a conclusion.